swayfmt/src/constants.rs

```diff
@@ -48,6 +48,11 @@ pub(crate) const CARRIAGE_RETURN: char = '\r';
 pub(crate) const WINDOWS_NEWLINE: &str = "\r\n";
 pub(crate) const UNIX_NEWLINE: &str = "\n";
 
+#[cfg(target_os = "windows")]
```

Comment:
These look like they are added also in #5178

Reply:
@JoshuaBatty yes, the first PR to be merged wins. The other will have a smaller diff.